cvpy/biomedimage/BiomedImage.py

```diff
@@ -78,45 +78,167 @@ def quantify_sphericity(self, image_table: ImageTable, use_spacing: bool, input_
+        >>> # Construct tables that are parameters to the quantify_sphericity API
         >>> image_table = s.CASTable(...)
         >>> input_table = ImageTable(image_table)
         >>> output_table = s.CASTable(...)
-        >>> ## Call the API
+        >>> # Call the API
         >>> BiomedImage.quantify_sphericity(input_table.table,....,output_table)
 
-        '''
+        """
         conn = self._cas_session
 
-        ## Quantify the volume and perimeter of the given component.
-        conn.biomedimage.quantifybiomedimages(images=dict(table= image_table.table),
+        # Quantify the volume and perimeter of the given component.
+        conn.biomedimage.quantifybiomedimages(images=dict(table=image_table.table),
                                               copyvars=['_path_'],
                                               region='COMPONENT',
                                               quantities=[dict(quantityparameters=dict(quantitytype='perimeter')),
-                                              dict(quantityparameters=dict(quantitytype='content',useSpacing = use_spacing))
-                                              ],
-                                              labelparameters=dict(labelType='basic',connectivity=label_connectivity.name), 
+                                                          dict(quantityparameters=dict(quantitytype='content',
+                                                                                       useSpacing=use_spacing))
+                                                          ],
+                                              labelparameters=dict(labelType='basic',
+                                                                   connectivity=label_connectivity.name),
                                               inputbackground=input_background,
                                               casout=dict(name='quantify'),
                                               )
 
-        ## Compute sphericity based on perimeter and volume of the lesion
+        # Compute sphericity based on perimeter and volume of the lesion
         conn.fedsql.execdirect(f'''
             create table {sphericity.name} as 
             select _path_,_perimeter_,_content_, (power(pi(), 1.0/3.0) * power(6*_content_, 2.0/3.0))/_perimeter_ as 
             sphericity from quantify
             ''')
-        
-        ## Delete the quantify table
+
+        # Delete the quantify table
         conn.table.dropTable(name='quantify')
 
-    
+    def mask_image(self, image: ImageTable, mask: ImageTable, casout: CASTable,
+                   input_background: int = 0, decode: bool = False, output_background: int = 0,
+                   add_columns: list(Enum) = None, copy_vars: list(Enum) = None):
+        """
+        Applies masking to an image table.
+
+        Parameters
+        ------------
+        image : ImageTable Object
```

Comment:
Instead of "ImageTable Object", you can markup the ImageTable class as :class:`ImageTable`. Aside from appearance, it makes it clickable with a link to the ImageTable API.

Reply:
Nit: Would you add a period to each description?

Reply:
Thanks for these suggestions, Kyle! I added periods and modified each of the type specifications. I don't believe we have Enums defined for columns yet so I took your list[enum.Enum] suggestion. Hopefully I formatted everything correctly but let me know if I didn't!